website/content/en/docs/about/concepts.md

```diff
@@ -54,6 +54,43 @@ A **pipeline** is a [directed acyclic graph][dag] of [components]. Each componen
 
 {{< jump "/docs/about/under-the-hood/architecture/pipeline-model" >}}
 
+
+## Buffers
+
+Sinks will try to send events as fast as possible. If they are unable to keep up, they have a configurable buffer that will hold events until they can be sent.
```

Comment:
Best to avoid future tense (here and elsewhere). In this case, "Sinks try to send events as fast as possible." The same goes for the rest of the doc.